app/javascript/spotlight/user/analytics.es6

```diff
@@ -1,9 +0,0 @@
-export default class {
-  connect() {
-    if (window._gaq != null) {
-      return _gaq.push(['_trackPageview']);
-    } else if (window.pageTracker != null) {
-      return pageTracker._trackPageview();
-    }
-  }
-}
```

Comment:
ga4 sends a pageview by default when it is initialized; no need to do this manually anymore. these two strategies correspond to earlier versions of analytics code.